src/course-unit/sidebar/index.jsx

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import classNames from 'classnames';
+import { Card, useToggle } from '@openedx/paragon';
+import { InfoOutline as InfoOutlineIcon } from '@openedx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import ModalNotification from '../../generic/modal-notification';
+import { editCourseUnitVisibilityAndData } from '../data/thunk';
+import { getCourseUnitData } from '../data/selectors';
+import { PUBLISH_TYPES } from '../constants';
+import { SidebarBody, SidebarFooter, SidebarHeader } from './components';
+import useCourseUnitData from './hooks';
+import messages from './messages';
+
+const Sidebar = ({ blockId, isDisplayUnitLocation, ...props }) => {
+  const {
+    title,
+    locationId,
+    releaseLabel,
+    visibilityState,
+    visibleToStaffOnly,
+  } = useCourseUnitData(useSelector(getCourseUnitData));
+  const intl = useIntl();
+  const dispatch = useDispatch();
+  const [isOpenDiscardModal, openDiscardModal, closeDiscardModal] = useToggle(false);
```

Comment:
```suggestion
  const [isDiscardModalOpen, openDiscardModal, closeDiscardModal] = useToggle(false);
```

Let's follow the codebase pattern, unless we want to change it across the board.

Reply:
Done